packages/celery-library/src/celery_library/worker/heartbeat.py

```diff
@@ -0,0 +1,23 @@
+import time
+from pathlib import Path
+
+HEARTBEAT_FILE = Path("/tmp/celery_heartbeat")  # noqa: S108
+
+
+def write_last_heartbeat() -> None:
+    tmp_file = HEARTBEAT_FILE.with_suffix(".tmp")
+    with tmp_file.open("w") as f:
+        f.write(f"{time.time()}")
+    tmp_file.replace(HEARTBEAT_FILE)
+
+
+def is_heartbeat_fresh(threshold_seconds: int = 10) -> bool:
```

Comment:
maybe this makes more sense?
```suggestion
def is_healthy(threshold_seconds: int = 10) -> bool:
```